client/parser/parser.go

```diff
@@ -10,13 +10,12 @@ import (
 	"alda.io/client/color"
 	"alda.io/client/help"
 	log "alda.io/client/logging"
-	model "alda.io/client/model"
+	"alda.io/client/model"
 )
 
 type parser struct {
 	filename string
 	input    []Token
-	updates  []model.ScoreUpdate
```

Comment:
@daveyarwood It doesn't look like `parser` needs to have this updates field so I've removed it.

Reply:
Interesting, looks like you're right! Maybe that was a holdover from an earlier implementation. Seems safe to remove.